src/openforms/submissions/api/serializers.py

```diff
@@ -259,7 +258,6 @@ def to_representation(self, instance):
         return super().to_representation(instance)
 
     def validate_data(self, data: dict):
-        validate_uploads(self.instance, data=data)
```

Comment:
We should now do one of two things:

* still call this if the feature flag to disable formio backend validation is enabled
* remove the feature flag to disable formio backend formio validation